spec/compute_spec.rb

```diff
@@ -10,16 +10,16 @@ def self.require(*_args); end
   describe "#new" do
     module Fog
       module TheRightWay
-        extend Provider
-        service(:compute, "Compute")
+        class Compute
+          def initialize(_args); end
+        end
       end
     end
 
     module Fog
       module TheRightWay
-        class Compute
-          def initialize(_args); end
-        end
+        extend Provider
+        service(:compute, "Compute")
```

Comment:
I don't get why you've changed the order here.
Is it because of some test failure?